kernel/safe_typing.mli

```diff
@@ -41,29 +41,20 @@ type 'a safe_transformer = safe_environment -> 'a * safe_environment
 
 (** {6 Stm machinery } *)
 
-type private_constant
 type private_constants
 
-type private_constant_role =
-  | Subproof
-  | Schema of inductive * string
-
 val side_effects_of_private_constants :
-  private_constants -> Entries.side_effect list
+  private_constants -> Entries.side_eff list
```

Comment:
Maybe this name change could be made unnecessary.

Reply:
The `side_eff` type corresponds to the previous one with the same name, while `side_effect` also contains the trusted environment. So, it's more that the `side_effects_of_private_constants` has changed type rather than the type has changed name.